Website/src/platform/CustomFields/DemoFieldTypeFactoryStore.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+using System.Collections.Generic;
+using Sitecore.Services.GraphQL.Content.TemplateGeneration.FieldMapping;
+using Sitecore.Services.GraphQL.EdgeSchema.GraphTypes.FieldTypes;
+using Sitecore.Services.GraphQL.EdgeSchema.TemplateGeneration.FieldMapping;
+
+namespace Sitecore.Demo.Edge.Website.CustomFields
+{
+    
+    public class DemoFieldTypeFactoryStore : IFieldTypeFactoryStore
+    {
+        private Dictionary<string, IFieldTypeFactory> _fieldTypes;
+
+        public DemoFieldTypeFactoryStore()
+        {
+            _fieldTypes = new Dictionary<string, IFieldTypeFactory>();
```

Comment:
Copying the code of an out of the box class seems risky. If we ever upgrade Sitecore/JSS/Headless, we will have to make sure we compare this file with the new default one.